problem_builder/tests/integration/test_titles.py

```diff
@@ -72,7 +72,7 @@ class StepTitlesTest(SeleniumXBlockTest):
     )
 
     mcq_template = """
-        <problem-builder mode="{{mode}}">
```

Comment:
This is unrelated, but made CircleCi tests to fail (strangely it passed on my system) --- ``{{`` is actually an escape sequence that means ``{`` in str.format.